### PR TITLE
Allow gaps on individual subsection as well

### PR DESCRIPTION
**Pull request by bmallred**
Issue: #1355

### CODE REVIEW

File: src/validators/people.js [hunk 1]
```diff
@@ -39,7 +96,7 @@ export default class PeopleValidator {
       }
     }
 
-    return true
+    return this.validCount() >= 3 && this.validYearRange()
```
Comment (toolness) [inline src/validators/people.js]: Should `3` be a named constant? I'm unclear on what its meaning is, but I'm also unfamiliar with the codebase.

File: src/validators/people.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ import AddressValidator from './address'
 import { validGenericTextfield, validPhoneNumber, validNotApplicable } from './helpers'
 import { decimalAdjust, rangeSorter, julian, findPercentage, today, daysAgo, julianNow } from '../components/Section/History/dateranges'
 
+const minimumYears = 7
+const minimumPeople = 3
```
Comment (toolness) [inline src/validators/people.js]: Thanks for making these constants and moving them up here!

An interesting thing I just realized is that when I'm reading code that's apart from a constant/variable's declaration, I tend to assume everything that's `IN_ALL_CAPS` is a constant, and that everything else is a variable. I assume you're using the above camelCase constant convention through your whole codebase though, so I don't think you should start changing your style just for this one file (or just for me).  Anyhow, just thought I'd mention it--you don't need to do anything about it, but I do find it useful to adopt some kind of convention that makes it easy for readers to tell constants apart from variables, I guess.

**Comment by floresj** [comment]
Gaps are now showing. Just one thing. If the user clicks on `Add Another Person` button,
selection_153

we populate it with the last day of the month. 

selection_152

Maybe we should set it to the current date.

**Comment by bmallred** [comment]
@floresj it would have to be conditional where if it exceeds the current date then we floor it to that. Otherwise we want to use the last day of the month in other circumstances.

**Pull request merged**